content/blog/en/version-6-dojo.md

```diff
@@ -0,0 +1,299 @@
+
+## Glob Support For Code Splitting
+
+The `.dojorc` configuration for `bundles` has been enhanced to support globs. The globs configuration option is especially useful for scenarios such as internationalization, meaning that you do not need to define all language bundles explicitly, instead simply define a matching pattern for each of the locales.
+
+Consider a project with a folder structure that defines language bundles in locale named directories. Ideally the build tool would create a single bundle for each locale that gets loaded when the user changes their locale. Using a glob provides a low maintenance and minimal effort way to do this without explicitly defining every language bundle module in the `.dojorc`.
+
+Prior to Dojo version 6, the configuration required to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ 
+        "src/widgets/home/nls/fr/home",
+        "src/widgets/menu/nls/fr/menu",
+        "src/widgets/blog/nls/fr/blog",
+        "src/widgets/reference/nls/fr/reference"
+      ],
+      "jp": [ 
+        "src/widgets/home/nls/jp/home",
+        "src/widgets/menu/nls/jp/menu",
+        "src/widgets/blog/nls/jp/blog",
+        "src/widgets/reference/nls/jp/reference"
+      ]
+    }
+  }
+}
+```
+
+Using the glob configuration in Dojo version 6 to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ "**/fr/*" ],
+      "jp": [ "**/jp/*" ]
+    }
+  }
+}
+```
+
+## Update to TypeScript
+
+The minimum required TypeScript version has been updated to 3.4. Updating the core framework to use a recent version enables us to leverage the latest features that underpin the function-based widget and middleware typings.
+
+## Revamped Doc Website
+
+For the Dojo 6 release, we have revamped [dojo.io](https://dojo.io) and moved it to a new repository, [dojo/site](https://github.com/dojo/site). Dojo.io now leverages the improved build time rendering (BTR) and block middleware. Documentation content is pulled down from [dojo/framework](https://github.com/dojo/framework) during the regular Dojo build process and rendered statically. The site is written completely using the new function-based widgets and middlewares, serving as a real world example of a statically rendered site using the latest Dojo patterns.
+
+## Migration
+
+As usual all of the breaking changes introduced in Dojo 6 are carefully considered, so that we truly believe the benefits outweigh the upgrade effort. To assist with the transition we have updated the CLI upgrade command, which will automatically upgrade your Dojo dependencies, upgrade your application code where possible and highlight areas in the application that require manual intervention. For more information on what has changed in Dojo 6, please see the [Dojo version 6 migration guide](https://github.com/dojo/framework/blob/master/docs/V6-Migration-Guide.md).
```

Comment:
```suggestion
As usual with modern Dojo releases,  all breaking changes introduced in Dojo 6 are carefully considered, so that we truly believe the benefits outweigh the upgrade effort. To assist with the transition we have updated the CLI upgrade command, which will automatically upgrade your Dojo dependencies, upgrade your application code where possible and highlight areas in the application that require manual intervention. For more information on what has changed in Dojo 6, please see the [Dojo version 6 migration guide](https://github.com/dojo/framework/blob/master/docs/V6-Migration-Guide.md).
```

Reply:
@dylans can you change the suggestion to remove the double space after `releases,`?